docs/pom.xml

```diff
@@ -46,6 +46,28 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>build-helper-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>com.github.sixro</groupId>
+                <artifactId>confluence-maven-plugin</artifactId>
+                <version>1.0.3</version>
+                <executions>
+                    <execution>
+                        <id>confluence-upload</id>
+                        <goals>
+                            <goal>generate</goal>
+                            <goal>deploy</goal>
+                        </goals>
+                        <phase>install</phase>
+                        <configuration>
+                            <serverId>ddf-confluence</serverId>
+                            <endpoint>http://tools.codice.org/wiki</endpoint>
+                            <!-- The wiki space docs will be uploaded too. Change this property on major releases -->
+                            <spaceKey>DDFTEM</spaceKey>
```

Comment:
Is it possible for spacekey to be a property that we can define elsewhere (e.g. ddf-parent)? This would allow us to change this potentially w/o having to do it in every repository.